test/WebJobs.Script.Tests/Configuration/ExtensionBundleOptionsSetupTest.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.BindingExtensionBundle;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ExtensionBundleOptionsSetupTest
+    {
+        private readonly TestEnvironment _environment = new TestEnvironment();
+        private readonly TestLoggerProvider _loggerProvider = new TestLoggerProvider();
+        private readonly string _hostJsonFile;
+        private readonly ScriptApplicationHostOptions _options;
+
+        public ExtensionBundleOptionsSetupTest()
+        {
+            string rootPath = Path.Combine(Environment.CurrentDirectory, "ScriptHostTests");
+
```

Comment:
Add test that mocks different environments, for example set AzureWebsiteInstanceId to mimic Azure hosted app.

Reply:
Added Environment Extensions tests in the PR follow up commit.